apps/core/src/components/affine/new-workspace-setting-detail/publish.tsx

```diff
@@ -100,7 +100,7 @@ const FakePublishPanelAffine = (_props: FakePublishPanelAffineProps) => {
     <Tooltip content={t['com.affine.settings.workspace.publish-tooltip']()}>
       <div className={style.fakeWrapper}>
         <SettingRow name={t['Publish']()} desc={t['Unpublished hint']()}>
-          <Switch checked={false} />
+          <Switch checked={false} onChange={() => {}} />
```

Comment:
```suggestion
          <Switch checked={false} onChange={noop} />
```

Reply:
Thanks for the feedback.
Updated to use `onChange={noop}`. @himself65  could you please take another look?